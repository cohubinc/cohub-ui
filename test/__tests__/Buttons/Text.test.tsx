import React from "react";
import { mount } from "enzyme";

import { Buttons } from "dist";

describe("Text", () => {
  it("renders without crashing", () => {
    const wrapper = mount(<Buttons.Text>push me</Buttons.Text>);
    expect(wrapper).toMatchSnapshot();
  });
});
