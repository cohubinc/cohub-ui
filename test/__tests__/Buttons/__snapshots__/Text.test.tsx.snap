// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders without crashing 1`] = `
<Text
  color="hsl(212, 14%, 35%)"
  fontSize="12px"
>
  <Blank
    className=" p-05"
    style={
      Object {
        "display": undefined,
      }
    }
  >
    <button
      className="Blank-module_ButtonBlank__2u_7K  p-05"
      style={
        Object {
          "display": undefined,
        }
      }
    >
      <Typography
        style={Object {}}
      >
        <Component
          style={Object {}}
        >
          <span
            className=""
            data-qa="text"
            style={
              Object {
                "color": "hsl(0, 2%, 22%)",
                "fontFamily": "Inter",
                "fontSize": "14px",
                "fontWeight": 400,
                "textAlign": "initial",
                "textTransform": "initial",
              }
            }
          >
            <Typography
              color="hsl(212, 14%, 35%)"
              kerning={0.07}
              style={
                Object {
                  "fontSize": "12px",
                }
              }
              uppercase={true}
              weight={500}
            >
              <Component
                color="hsl(212, 14%, 35%)"
                kerning={0.07}
                style={
                  Object {
                    "fontSize": "12px",
                  }
                }
                uppercase={true}
                weight={500}
              >
                <span
                  className=""
                  data-qa="text"
                  style={
                    Object {
                      "color": "hsl(212, 14%, 35%)",
                      "fontFamily": "Inter",
                      "fontSize": "12px",
                      "fontWeight": 500,
                      "letterSpacing": "0.07rem",
                      "textAlign": "initial",
                      "textTransform": "uppercase",
                    }
                  }
                >
                  push me
                </span>
              </Component>
            </Typography>
          </span>
        </Component>
      </Typography>
    </button>
  </Blank>
</Text>
`;
